extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -155,5 +164,33 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		onAdLoadCallback(element.getId(), gptEvent, iframe);
 	};
 
+	GoogleTag.prototype.destroySlots = function (slotsNames) {
+		var slotsToDestroy = [], success;
+
+		slotsNames.forEach(function (slotName) {
+			if (slotsMap[slotName]) {
+				slotsToDestroy = slotsToDestroy.concat(slotsMap[slotName]);
+			}
+		});
+
+		console.log('bogna', 'about to destroy slots', slotsToDestroy);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2228)